content/conformance-logos.md

```diff
@@ -165,35 +212,31 @@ Below are the logos as they will appear in your site. You may also choose to dow
+      <td><img src="https://www.w3.org/WAI/wcag2AA-blue" alt="WCAG 2.0 AA (blue)" width="88" height="31" /><br />
+        (<a href="https://www.w3.org/WAI/wcag2AA-blue.png" title="PNG version of WCAG 2.0 Level AA conformance icon (blue)">png</a>, <a href="https://www.w3.org/WAI/wcag2AA-blue.gif" title="GIF version of WCAG 2.0 Level AA conformance icon (blue)">gif</a>, <a href="https://www.w3.org/WAI/wcag2AA-blue-v.svg" title="SVG version of WCAG 2.0 Level AA conformance icon (blue)">svg</a>, <a href="https://www.w3.org/WAI/wcag2AA-blue-v.eps" title="EPS version of WCAG 2.0 Level AA conformance icon (blue)">eps</a>)</td>
+      <td><img src="https://www.w3.org/WAI/wcag2AAA-blue" alt="Level AAA conformance icon, W3C-WAI Web Content Accessibility Guidelines 2.0 (blue)" width="88" height="31" /><br />
+        (<a href="https://www.w3.org/WAI/wcag2AAA-blue.png" title="PNG version of WCAG 2.0 Level AAA conformance icon (blue)">png</a>, <a href="https://www.w3.org/WAI/wcag2AAA-blue.gif" title="GIF version of WCAG 2.0 Level AAA conformance icon (blue)">gif</a>, <a href="https://www.w3.org/WAI/wcag2AAA-blue-v.svg" title="SVG version of WCAG 2.0 Level AAA conformance icon (blue)">svg</a>, <a href="https://www.w3.org/WAI/wcag2AAA-blue-v.eps" title="EPS version of WCAG 2.0 Level AAA conformance icon (blue)">eps</a>)</td>
     </tr>
   </tbody>
 </table>
 
-## Scope of the claim {#scope}
-
-By default, a conformance icon refers to a single page. If the claim is meant to apply to include more than one page, the conformance icon must be accompanied by explicit scope information explaining which pages are covered by the claim.
-
 ## Responsibility for accuracy of claims {#responsibility}
 
-Content providers are solely responsible for the use of these logos. Before using these logos as part of a conformance claim, we recommend that the provider be familiar with the [Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG20/) and use a variety of review methods to ensure that any page using this logo meets the conformance level claimed. Providers should also ensure that anyone maintaining or updating the site is familiar with logo use, and either re-reviews the page or removes the logo from the page if they are unsure whether it still meets a specified conformance level.
+Content providers are solely responsible for the use of these logos. Before using these logos as part of a conformance claim, we recommend that the provider understand [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/standards-guidelines/wcag/) and sufficiently [evaluate](https://www.w3.org/WAI/test-evaluate/conformance/) to ensure that pages using this logo meet the conformance level claimed. Providers should also ensure that anyone maintaining or updating the site is familiar with WCAG and the logo use, and either re-reviews the page or removes the logo from the page if they are unsure whether it still meets a specified conformance level.
```

Comment:
I this it should be "understand **the** Web Content..."

and "sufficiently evaluate **the pages** to ensure they meet the conformance level claimed"

Reply:
> I this it should be "understand the Web Content..."

I updated the Style Guide to clarify "understand WCAG 2" https://www.w3.org/WAI/EO/wiki/Style#Referring_to_xxAG_.28WCAG_2.3B_singular.29 -- and also tweaked the wording so it flows better.

> and "sufficiently evaluate the pages to ensure they meet the conformance level claimed"

I'm a little hesitant to put even more emphasis on "pages". I tried this

> ... provider understand... (WCAG) 2 and sufficiently [evaluate accessibility](https://www.w3.org/WAI/test-evaluate/conformance/) to ensure that pages using this logo meet the conformance level claimed.

What do you think?